### ISSUE TITLE
Quota filled and NaN file size

### ISSUE THREAD

**Issue by ff98li** [codalab/codabench#1388]
The quota filled issue has been mentioned in https://github.com/codalab/codabench/issues/1364 but I see no one mentions the file sizes being NaN so I would like to bring this to the developers and admins' attention. This issue still persists.

![image](https://github.com/codalab/codabench/assets/55061801/f5499c66-46b9-47e4-8604-fa0ef30dd79d)

I am not sure if it has to do with the competition we are managing receiving too many submissions that cause this problem, but mine and the other adminstrators' accounts all have the same issue. Currently we are unable to upload any files to the platform. If anyone here in the community who have experience in hosting challenges on Codabench and have experienced similar problems, any insights will be very much appreciated.

**Comment by ihsaan-ullah** [comment]
When you upload a submission what error do you get?
Can you share your competition link? If you don't want to share the link here, you can email us here: `info@codalab.org`. This is a private email monitored by codalab/codabench administrators only

**Comment by ihsaan-ullah** [comment]
You haven't shared the error you are getting when you upload a file.

**Comment by ff98li** [comment]
Hi @ihsaan-ullah ,
thank you very much for the quick response! I have sent an email with the link to our competition page. Currently not just data or program files in the Resource page, submission files can't be uploaded via the competition page either:
![image](https://github.com/codalab/codabench/assets/55061801/b23bd63d-0e8f-4c8e-86df-030c55bb56fa)
Clicking the "add attanchment" button shows neither error messages nor anything else. It simply doesn't respond.

**Comment by ihsaan-ullah** [comment]
I have experienced this problem using Chrome. Can you please try another browser to see if the problem is still there.

**Comment by ff98li** [comment]
Hi @ihsaan-ullah ! Thank you for the advice. Switching browsers solves the file upload problem. Should I keep this issue open for the `NaN` issue or close it?

**Comment by ihsaan-ullah** [comment]
> Hi @ihsaan-ullah ! Thank you for the advice. Switching browsers solves the file upload problem. Should I keep this issue open for the `NaN` issue or close it?

Yes, please keep it open so that we can keep track of the NAN issue. Thank you for reporting the problems.

**Comment by ihsaan-ullah** [comment]
@ff98li can you try to do the following to see if you can regenerate `NAN`

1. upload any zip file as ingestion program
2. Hard refresh (Mac: Command+Shift+R) and reupload the same file again

**Comment by ff98li** [comment]
> @ff98li can you try to do the following to see if you can regenerate `NAN`
> 
> 1. upload any zip file as ingestion program
> 2. Hard refresh (Mac: Command+Shift+R) and reupload the same file again

<img width="1106" alt="Screenshot 2024-04-11 at 9 22 50 PM" src="https://github.com/codalab/codabench/assets/55061801/9d20e2a5-3548-4a36-867b-6190b4865a47">

Hi @ihsaan-ullah , I just tried reproducing the issue following the steps, but `NaN` doesn't come up. Any suggestions?

**Comment by ihsaan-ullah** [comment]
That is strange. Even if you upload the same files which previously gave you NAN?

**Comment by ff98li** [comment]
> That is strange. Even if you upload the same files which previously gave you NAN?

Yea. The latest uploaded files all look good, even if they are the same files previously uploaded showing NaN. I think only the previously uploaded files don't show the file size properly (NaN). The new ones are good.

**Comment by Didayolo** [comment]
Maybe we had a temporary problem on the storage or on our configuration that is now solved? This is quite hard to investigate, at least it is nice that it's now working fine. We should keep an eye on this.

**Comment by ihsaan-ullah** [comment]
> Maybe we had a temporary problem on the storage or on our configuration that is now solved? This is quite hard to investigate, at least it is nice that it's now working fine. We should keep an eye on this.

We may want to close this but keep this issue mentioned for Tristan

**Comment by ff98li** [comment]
> > Maybe we had a temporary problem on the storage or on our configuration that is now solved? This is quite hard to investigate, at least it is nice that it's now working fine. We should keep an eye on this.
> 
> We may want to close this but keep this issue mentioned for Tristan

Yea it seems to be a temporary problem. Since things are working just fine now I am cool with closing it.
